gadopt/gplates/gplatesfiles.py

```diff
@@ -1,33 +1,47 @@
 from pathlib import Path
 
-
-def obtain_Muller_2022_SE(base_path: str | Path):
-
-    plate_reconstruction_files = {
-        "rotation_filenames": ["optimisation/1000_0_rotfile_MantleOptimised.rot"],
-        "topology_filenames": [
-            "250-0_plate_boundaries.gpml",
-            "410-250_plate_boundaries.gpml",
-            "1000-410-Convergence.gpml",
-            "1000-410-Divergence.gpml",
-            "1000-410-Topologies.gpml",
-            "1000-410-Transforms.gpml"
-        ]
-    }
-
-    # This is the path to download and extract files
-    base_path = Path(base_path)
-    base_path = base_path / "Muller_etal_2022_SE_1Ga_Opt_PlateMotionModel_v1.2"
-
-    # Check if all files are already present
-    all_files_present = all((base_path / filename).exists() for filenames in plate_reconstruction_files.values() for filename in filenames)
+_default_plate_files = {
```

Comment:
Since this only works for Muller et al, I suggest changing the name for this:
```suggestion
_default_muller2018_plate_files = {